cypher/cypher-docs/src/docs/dev/ql/administration/constraints/list-constraints-table-columns.asciidoc

```diff
@@ -0,0 +1,53 @@
+.List constraints output
+[options="header", width="100%", cols="1a,4,^.^,^"]
+|===
+| Column
+| Description
+| Brief output
+| Verbose output
+
+| id
+| The id of the constraint.
+| `+`
+| `+`
+
+| name
+| Name of the constraint (explicitly set by the user or automatically assigned).
+| `+`
+| `+`
+
+| type
+| The ConstraintType of this constraint (`UNIQUENESS`, `NODE_PROPERTY_EXISTENCE`, `NODE_KEY`, or `RELATIONSHIP_PROPERTY_EXISTENCE`).
+| `+`
+| `+`
+
+| entityType
+| Type of entities this constraint represents (nodes or relationship).
+| `+`
+| `+`
+
+| labelsOrTypes
+| The labels or relationship types of this constraint.
+| `+`
+| `+`
+
+| properties
+| The properties of this constraint.
+| `+`
+| `+`
+
+| ownedIndexId
+| The id of the index associated to the constraint.
+| `+`
+| `+`
+
+| options
+| The options passed to `CREATE` command.
+|
+| `+`
```

Comment:
I think we should mention that these are null for existence constraints

Reply:
yes